frontend/test/cypress/support/commands.ts

```diff
@@ -23,4 +23,22 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+/// <reference path="../support/index.d.ts" />
 import '@testing-library/cypress/add-commands';
+
+Cypress.Commands.add('isWithinViewport', { prevSubject: true }, (element: any) => {
+  const windowInnerWidth = Cypress.config(`viewportWidth`);
```

Comment:
Why `Cypress.config('viewportWidth')` instead of `Cypress.$(cy.state('window')).height();`? I'm guessing if a test manually changes the viewport then the value in the config won't be accurate?

Reply:
>  I'm guessing if a test manually changes the viewport then the value in the config won't be accurate?

Thank you for pointing this out and you were right! Fixed it to use the current window width/height in https://github.com/iFixit/react-commerce/pull/381/commits/f71029468b10af5103d2eb5646945d04cfe69fcf.